firmware/controllers/settings.cpp

```diff
@@ -1169,7 +1160,8 @@ typedef struct {
 } command_f_s;
 
 const command_f_s commandsF[] = {
-#if EFI_ENGINE_CONTROL && EFI_ENABLE_MOCK_ADC
+#if EFI_ENGINE_CONTROL
```

Comment:
this looks a bit confusing, I would prefer to separate blocks
if A&B
endif
if B
endif